sample/build.gradle.kts

```diff
@@ -19,8 +19,8 @@ dependencies {
   implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.0")
   implementation("org.jetbrains.kotlinx:kotlinx-datetime:0.2.1")
 
-  implementation(enforcedPlatform("org.jetbrains.kotlin-wrappers:kotlin-wrappers-bom:0.0.1-pre.256-kotlin-1.5.31"))
-  implementation("org.jetbrains.kotlin-wrappers:kotlin-react-dom")
+  implementation(enforcedPlatform("org.jetbrains.kotlin-wrappers:kotlin-wrappers-bom:0.0.1-pre.286-kotlin-1.6.10"))
+  implementation("org.jetbrains.kotlin-wrappers:kotlin-react-dom-legacy")
```

Comment:
If this will eventually go away, would be great to switch to the non-legacy artifact in the sample. But that can be a follow up PR. I like keeping this PR just about compatibility.

Reply:
[Here](https://github.com/JetBrains/kotlin-wrappers/blob/master/CHANGELOG.md) the new changes in kotlin-wrappers

Maybe there will need to be a rewrite of the library to support the new syntax.